packages/IMAPClient-Core.package/ICEndPoint.class/instance/parseFetchFlags.for..st

```diff
@@ -0,0 +1,13 @@
+protocol parse
+parseFetchFlags: aHeaderChunk for: anEmail
+
+	((aHeaderChunk at: 1) includesSubstring: '\Seen')
```

Comment:
I would suggest: `anEmail seen: ((aHeaderChunk at: 1) includesSubstring: '\Seen')`. No need for the conditional logic :)